src/texture/buffer_texture.rs

```diff
@@ -550,6 +550,11 @@ impl<'a> TextureExt for BufferTextureRef<'a> {
         unsafe { ctxt.gl.BindTexture(gl::TEXTURE_BUFFER, self.texture); }
         gl::TEXTURE_BUFFER
     }
+
+    fn prepare_for_access(&self, _: &mut CommandContext<'_>, access_type: crate::TextureAccess) {
+        // TODO: Right now this type of texture cannot be used in an image unit
```

Comment:
I wonder if a panic or some other error would be better here?

Reply:
I think it is better to leave this empty, as this type cannot be made into an image unit at the moment, therefore the missing implementation has no adverse effect.

If we added a panic here it would make it impossible to use this type as a shader uniform, as it would always result in a panic.